sources/web/datalab/server.ts

```diff
@@ -94,14 +96,6 @@ function handleJupyterRequest(request: http.ServerRequest, response: http.Server
 function handleRequest(request: http.ServerRequest,
                        response: http.ServerResponse,
                        path: string) {
-  // TODO(jupyter): Additional custom path - should go away eventually with replaced
-  // pages.
-  // /static and /custom paths for returning static content
-  if ((path.indexOf('/static') == 0) || (path.indexOf('/custom') == 0)) {
```

Comment:
I don't think these will be logged; handling of static and custom paths is being moved earlier, not later.

More specifically, the 'uncheckedRequestHandler' method will now handle those requests, and it is the caller of this method, so those requests will already be filtered out before this method is called.